pkg/controller/controller_instance.go

```diff
@@ -1843,7 +1843,12 @@ func (c *controller) serviceInstanceHasExistingBindings(instance *v1beta1.Servic
 	}
 	for _, binding := range bindingList {
 		if instance.Name == binding.Spec.ServiceInstanceRef.Name {
-			return true, nil
+			for _, condition := range binding.Status.Conditions {
+				if condition.Type == v1beta1.ServiceBindingConditionReady && condition.Status != v1beta1.ConditionFalse {
+					return true, nil
```

Comment:
I don't remember - what will happen when the user tries to delete the binding later? If fails b/c can't find the instance, I'd rather see that fixed in the same PR

Reply:
@arschles If the binding has its `UnbindStatus` as `StatusNotRequired` (aka. there was never a request sent to the broker), it will be successfully deleted without having to look for the instance.